package.json

```diff
@@ -43,8 +43,8 @@
     "publish:beta": "yarn publish --tag=beta"
   },
   "dependencies": {
-    "@pedrouid/iso-crypto": "^1.1.0",
-    "@stablelib/x25519": "^1.0.0",
+    "@pedrouid/iso-crypto": "https://github.com/jmrossy/iso-crypto#452eef4",
```

Comment:
TODO, replace this with iso-crypto's `2.0.0-beta.1` version when that's released.